content/fr/_index.md

```diff
@@ -22,7 +22,7 @@ Datadog recueille l'ensemble de ces métriques, événements et états de servic
 
 ## Collecte exhaustive
 
-Recueillez une multitude de données déjà disponibles sans écrire de code. [Installez l'Agent Datadog][1] où bon vous semble : sur vos serveurs, instances, machines virtuelles, nœuds et [hosts exécutant des conteneurs][2]. Activez ensuite l'Agent et configurez l'une de nos 350 [intégrations][3] prêtes à l'emploi pour commencer à envoyer vos métriques au backend de Datadog.
+Recueillez une multitude de données déjà disponibles sans écrire de code. [Installez l'Agent Datadog][1] où bon vous semble : sur vos serveurs, instances, machines virtuelles, nœuds et [hosts exécutant des conteneurs][2]. Activez ensuite l'Agent et configurez l'une de nos 200 [intégrations][3] prêtes à l'emploi pour commencer à envoyer vos métriques au backend de Datadog.
```

Comment:
@davidejones looks like changes to the Integrations numbers that went up yesterday were reverted by this PR. Any idea why?

Reply:
I think its because the change was made directly to the `fr` files and i happened to have the bidirectional code in "dryrun" mode meaning it didn't send the changes it only faked it.

I'll make sure that's switched on tomorrow and test it again, i had been turning it on and off while i fixed issues.

Reply:
@davidejones - Do I need to submit this update again?

Reply:
Thanks @ruthnaebeck I think we should be good though